openedx/features/edly/utils.py

```diff
@@ -108,3 +108,28 @@ def get_enabled_organizations(request):
         return []
 
     return [studio_site_edx_organization]
+
+
+def create_edly_user_profile(request, user):
+    """
+    Create edly user profile for User.
+
+    Arguments:
+        request: HTTP request object.
+        user: User object.
+
+    Returns:
+        object: EdlyUserProfile object.
+    """
+
+    current_site = request.site
+
+    edly_sub_org = EdlySubOrganization.objects.get(
+        Q(lms_site=current_site) |
+        Q(studio_site=current_site)
+    )
```

Comment:
Why aren't we getting edly sub org from site? like `edly_sub_org = current_site.edly_sub_org_for_lms`

Reply:
Yes. I believe the user registration is possible on LMS only.

Reply:
Done.